test/Microsoft.ML.Predictor.Tests/TestTransposer.cs

```diff
@@ -199,9 +199,9 @@ public void TransposerTest()
             using (Transposer trans = Transposer.Create(Env, view, true, 3, 5, 4))
             {
                 // Check to see that A, B, and C were not transposed somehow.
-                Assert.Null(trans.TransposeSchema.GetSlotType(0));
-                Assert.Null(trans.TransposeSchema.GetSlotType(1));
-                Assert.Null(trans.TransposeSchema.GetSlotType(2));
+                Assert.Null((trans as ITransposeDataView).TransposeSlotTypes[0]);
```

Comment:
Can you just do the cast 1 time instead of 3 times in a row?

Reply:
We have only one `(ITransposeDataView)trans` now.

---
In reply to: [248704226](https://github.com/dotnet/machinelearning/pull/2111#discussion_r248704226) [](ancestors = 248704226)